.github/workflows/build.yaml

```
@@ -5,7 +5,30 @@ on:
       - main
   pull_request:
     types: [opened, synchronize, reopened]
+
 jobs:
+  test:
+    name: Run Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v2
```

actionlint

the runner of "actions/setup-python@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)